benchopt/cli/helpers.py

```diff
@@ -70,6 +77,46 @@ def clean(benchmark, token=None, filename='all'):
     benchmark.mem.clear(warn=False)
 
 
+def clean_archive(info):
+    if "__pycache__" in info.name:
+        return None
+
+    # reset the username and info in the archive
+    info.uid = info.gid = 0
+    info.uname = info.gname = "benchopt"
+
+    return info
+
+
+@helpers.command(
+    help="Create an archive of the benchmark that can easily be shared."
+)
+@click.argument('benchmark', default=Path.cwd(), type=click.Path(exists=True),
+                shell_complete=complete_benchmarks)
+@click.option('--with-outputs',
+              is_flag=True,
+              help="If this flag is set, also store the outputs of the "
+              "benchmark in the archive.")
+def archive(benchmark, with_outputs):
+
+    benchmark = Benchmark(benchmark)
+    bench_dir = benchmark.benchmark_dir
+
+    to_archive = ARCHIVE_ELEMENTS
+    if with_outputs:
+        to_archive = to_archive + ['outputs/*']
+
+    archive_name = f"{benchmark.name}.tar.gz"
+    print(f"Creating {archive_name}...", end='', flush=True)
```

Comment:
This gives me:
```
(base) ➜  lasso git:(main) ✗ benchopt archive
Creating lasso.tar.gz.../home/mathurin/workspace/benchs/lasso/README.rst
/home/mathurin/workspace/benchs/lasso/example_config.yml
```
we can have a regular print I think

and maybe there is no need to print every file being put inside the archive. no strong opinion here